content/channels/index.textile

```diff
@@ -447,6 +447,72 @@ To publish a single message to multiple channels, make multiple "@publish()@":/a
+_*Solution:*_ Implementing server-side batching allows the platform to group multiple chat messages and reactions into single batches before delivering them to subscribers. This reduces the number of individual messages processed and helps maintain a seamless and interactive user experience during high-activity periods.
+
+* *Fan engagement platforms:*
+A sports fan engagement app allows users to send reactions, such as likes and comments, during live matches. During key moments, like a goal being scored, the app faces a surge in user interactions, leading to a spike in message volume.
+_*Solution:*_ Server-side batching can combine multiple reactions into a batch before sending them to subscribers. This helps manage the high volume of interactions efficiently, preventing rate limits from being exceeded and ensuring all reactions are delivered promptly.
+
+* *Cursor tracking:*
+A collaborative workspace application tracks user presence and cursor movements in realtime, showing the live cursor positions of multiple users. Continuous updates on cursor positions from numerous users can generate a high volume of messages, increasing costs and potentially hitting rate limits.
+_*Solution:*_ The application can batch these frequent updates into single messages by applying server-side batching to the cursor and presence messages. This reduces the number of messages processed and sent, lowering costs and maintaining realtime tracking accuracy.
+
+h4. How server-side batching works
+
+The server-side batching process works through a *batching policy* and a *batching interval:*
+
+h4. Batching policy
+
+The batching policy defines the rules and behaviors for aggregating messages during the batching interval. The initial release supports the *simple* policy, which groups multiple incoming messages into a single batch without altering their content or metadata.
+
+A batch can contain up to 50 messages by count or total data size. For example, if you have 60 messages, they will be split into two batches: one with 50 messages and another with 10. If the combined data size of 50 messages exceeds a specific bandwidth threshold _e.g. in kilobytes_ the batch size will be limited to that threshold.
+
+h4. Batching interval
+
+The batching interval is a configurable period specified in milliseconds _ranging from 20ms to 1s_, during which incoming messages are collected and grouped into a batch. Messages sent to the server during this interval are temporarily held and aggregated. Once the interval elapses, the collected messages are combined into a single batch and delivered to subscribers as one message.
+
+h4. Choose the right interval for your app
+
+* Shorter intervals benefit real-time applications.
```

Comment:
I'd assume that 99% of apps are realtime if they're using Ably 🙂

Reply:
Removed